src/components/theme/ThemeValueContext.ts

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
+import {RealTheme} from './types';
 
 export interface ThemeValueContextProps {
-    themeValue: string;
+    themeValue: RealTheme;
 }
 
 const initialValue: ThemeValueContextProps = {
-    themeValue: '',
+    themeValue: 'light',
```

Comment:
Looks like there should be `DEFAULT_THEME` from constants

Reply:
`DEFAULT_THEME` is `'system'`, theme value should be a "real" theme